lib/CodeTransform/Tests/Adapter/WorseReflection/Refactor/WorseExtractMethodTest.php

```diff
@@ -39,154 +35,69 @@ public function testExtractMethod(string $test, string $name, ?string $expectedE
-                'newMethod',
-            ],
-            'adds return type for class' => [
-                'extractMethod17.test',
-                'newMethod',
-            ],
-            'extracts expression to method' => [
-                'extractMethod18.test',
-                'newMethod',
-            ],
-            'extracts assignment expression to method' => [
-                'extractMethod19.test',
-                'newMethod',
-            ],
-            'extracts assignment expression with unknown return type' => [
-                'extractMethod20.test',
-                'newMethod',
-            ],
-            'extract expression and adds short return type for class' => [
-                'extractMethod21.test',
-                'newMethod',
-            ],
-            'return if extracted code has a return' => [
-                'extractMethod22.test',
-                'newMethod',
-            ],
-            'adds method to declaring class' => [
-                'extractMethod23.test',
-                'newMethod',
-            ],
-            'different scopes 1' => [
-                'extractMethod24.test',
-                'newMethod',
-                'Cannot extract method. Check if start and end statements are in different scopes.'
-            ],
-            'different scopes 2' => [
-                'extractMethod25.test',
-                'newMethod',
-                'Cannot extract method. Check if start and end statements are in different scopes.'
-            ],
-            'different scopes 3' => [
-                'extractMethod26.test',
-                'newMethod',
-                'Cannot extract method. Check if start and end statements are in different scopes.'
-            ],
-            'empty text selection' => [
-                'extractMethod27.test',
-                'newMethod',
-            ],
-            'empty selection 2' => [
-                'extractMethod28.test',
-                'newMethod',
-                'Cannot extract method. Check if start and end statements are in different scopes.'
-            ],
-            'nullable argument' => [
-                'extractMethod29.test',
-                'newMethod',
-            ],
-            'ignore scoped variables: catch clause' => [
-                'extractMethod30.test',
-                'newMethod',
-            ],
-            'ignore scoped variables: anonymous function' => [
-                'extractMethod31.test',
-                'newMethod',
-            ],
-            'union argument' => [
-                'extractMethod32.test',
-                'newMethod',
-            ],
-            'extract method from trait' => [
-                'extractMethod33.test',
-                'newMethod'
-            ],
+        yield['extractMethod24.test'];
+        yield['extractMethod25.test'];
+        yield['extractMethod26.test'];
+        yield 'empty selection 2' => [
+            'extractMethod28.test',
         ];
```

Comment:
not sure i like this. the description has been lost, I'd rather either:

1) Keep the old test style
2) Refactor to `yield 'union argument => [ '...' ]`

I don't really mind if we keep or remove the method name.